src/Microsoft.AspNetCore.Html.Abstractions/HtmlEncodedString.cs

```diff
@@ -17,17 +17,17 @@ public class HtmlEncodedString : IHtmlContent
         /// </summary>
         public static readonly IHtmlContent NewLine = new HtmlEncodedString(Environment.NewLine);
 
-        private readonly string _value;
-
         /// <summary>
         /// Creates a new <see cref="HtmlEncodedString"/>.
         /// </summary>
         /// <param name="value">The HTML encoded value.</param>
         public HtmlEncodedString(string value)
         {
-            _value = value;
+            Value = value;
         }
 
+        public string Value { get; }
```

Comment:
Unrelated to my specific changes, but I used this to make an optimization in Razor where I can avoid escaping the content inside an attribute when the string doesn't contain `"`